exasol/analytics/audit/columns.py

```diff
@@ -0,0 +1,50 @@
+from exasol.analytics.schema import (
+    decimal_column,
+    timestamp_column,
+    varchar_column,
+)
+
+
+class BaseAuditColumns:
```

Comment:
I would write the comments as docstrings.

Reply:
Sigh. Long discussion with Torsten ended with "let's use source code comments".